Setup/Product.wxs

```diff
@@ -1,393 +1,396 @@
+                <File Source="..\Externals\Git.hub\Git.hub\bin\$(var.Configuration)\Git.hub.dll" />
+            </Component>
+            <Component Id="PSTaskDialog.dll" Guid="*">
+                <File Source="..\bin\PSTaskDialog.dll" />
+            </Component>
+            <Component Id="RestSharp.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\RestSharp.dll" />
+            </Component>
+            <Component Id="Microsoft.VisualStudio.Threading.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\Microsoft.VisualStudio.Threading.dll" />
+            </Component>
+            <Component Id="Microsoft.VisualStudio.Validation.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\Microsoft.VisualStudio.Validation.dll" />
+            </Component>
+            <Component Id="Microsoft.WindowsAPICodePack.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\Microsoft.WindowsAPICodePack.dll" />
+            </Component>
+            <Component Id="Microsoft.WindowsAPICodePack.Shell.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\Microsoft.WindowsAPICodePack.Shell.dll" />
+            </Component>
+            <Component Id="NBug.dll" Guid="*">
+                <File Source="..\Externals\NBug\NBug\bin\$(var.Configuration)\NBug.dll" />
+            </Component>
+            <Component Id="SmartFormat.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\SmartFormat.dll" />
+            </Component>
+            <Component Id="System.Reactive.Core.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Reactive.Core.dll" />
+            </Component>
+            <Component Id="System.Reactive.Interfaces.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Reactive.Interfaces.dll" />
+            </Component>
+            <Component Id="System.Reactive.Linq.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Reactive.Linq.dll" />
+            </Component>
+            <Component Id="System.Reactive.PlatformServices.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Reactive.PlatformServices.dll" />
+            </Component>
+            <Component Id="System.ValueTuple.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\System.ValueTuple.dll" />
+            </Component>
+            <Component Id="System.Runtime.InteropServices.RuntimeInformation.dll" Guid="*">
+                <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Runtime.InteropServices.RuntimeInformation.dll" />
+            </Component>
+            <Component Id="Microsoft.VisualStudio.Composition.dll" Guid="*">
```

Comment:
A minor nitpick: this component definition could be moved next to the other two Microsoft.VisualStudio.* components.

Reply:
Will do after I deal with the System.Collections.Immutable mess.

Reply:
The issue is Jira plugin somehow has a different version of immutable dll.
![image](https://user-images.githubusercontent.com/3629489/50539857-78e65f00-0b55-11e9-9a12-dce79361d348.png)

Since it builds after our base plugin library we have a dependency issue.  Running prepare release script, the portable build gives me 
![image](https://user-images.githubusercontent.com/3629489/50539825-f1005500-0b54-11e9-94e7-5cb1189691d2.png)
![image](https://user-images.githubusercontent.com/3629489/50539870-9f0bff00-0b55-11e9-995b-8434ea470980.png)

while all the other copies in the bin folders of other projects besides jira have this one
![image](https://user-images.githubusercontent.com/3629489/50539841-2efd7900-0b55-11e9-837f-749c5185ba68.png)
![image](https://user-images.githubusercontent.com/3629489/50539885-d1b5f780-0b55-11e9-8776-cf79152b90c4.png)

Tracking down which nuget and maybe looking into adding a specific nuget version to all projects.